python/processors/keep_and_drop.txt

```diff
@@ -12,7 +12,7 @@ drop fixedGridRho*
 drop Electron_mva*
 # drop *btagCSVV2*
 drop *btagCMVA*
-drop FatJet_deepTagMD_*
+#drop FatJet_deepTagMD_*
```

Comment:
Do we need MD tagger?

Reply:
This is for studies being done by Ken and Bryan for the ttZ analysis.